### ISSUE TITLE
pypy for performance tests

### ISSUE THREAD

**Issue by goto40** [textX/textX#227]
Just an idea:

I used textx for a simple parsing task for large files. The whole procedure was very slow, so I switched to pypy (https://pypy.org/). This gave me a big speedup (x2,x3 or even faster).

We could try to include pypy for our automated tests.

**Comment by igordejanovic** [comment]
Yes, pypy is great. I had similar experiences in the past. I agree that we should support it if possible.
